masonite/providers/HelpersProvider.py

```diff
@@ -25,6 +26,7 @@ def boot(self, view: View, request: Request):
         builtins.env = os.getenv
         builtins.resolve = self.app.resolve
         builtins.route = request.route
+        builtins.mail = manager.helper
```

Comment:
you know .. after looking at this code I think we're going to have some naming conflicts with the mail module in the project .. Let's name the mail helper `mail_helper()` instead .. 🤔 

sort of ugly but because of how Python manages global namespaces ..